test/config/e2e-tear-down.js

```diff
@@ -4,16 +4,5 @@ import setup from './e2e-setup';
 describe('tear down', function() {
   const client = setup();
 
-  after(async () => {
-    try {
-      await client.end();
-      console.log('Successfully Ended Webriver Session');
-      process.exit(0);
-    } catch (err) {
-      console.log('Error Ending Webriver Session', err.stack);
-      process.exit(1);
-    }
-  });
-
   it('ends the webdriver session', () => {});
```

Comment:
@stephenkao I was using the `process.exit` to avoid `pkill java`....is that not a problem anymore?